src/Curiosity/curiosity.tsx

```diff
@@ -0,0 +1,35 @@
+  
+
+import React from 'react';
+import {useState, useEffect} from "react";
+import {BrowserRouter as Router, Switch, Route, Link} from 'react-router-dom';
+import './curiosity.scss';
+
+function Curiosity() {  
+    const [myPhotos, setMyPhotos] = useState(null);
+    useEffect(() => {
+        fetch("https://api.nasa.gov/mars-photos/api/v1/rovers/curiosity/photos?earth_date=2021-2-15&camera=navcam&api_key=HX6J0TTAmtm3sivr4NNfbdf1o3GebnOtLXyvGJWB")
```

Comment:
As you are the first group to submit a PR with an API_KEY, we'll need to think about where we want to store an API key. We won't want to push our API keys to github and have them public (although these ones aren't very secret from NASA they are only used for rate limiting!) so we'll want a way to be able to use an API_KEY locally while not actually committing them to the project. I would suggest creating a .env file in the root folder, adding the API key there and using it as a variable in the code, and adding .env files to your .gitignore file so that it is not committed to the github repo. We should then update the ReadMe to explain that everyone will need to create their own .env file with their own API key

Reply:
(Feel free to grab me to go over this!)

Reply:
https://create-react-app.dev/docs/adding-custom-environment-variables/ this link will be helpful for the above as it outlines what we are trying to do which is add an environment variable :)